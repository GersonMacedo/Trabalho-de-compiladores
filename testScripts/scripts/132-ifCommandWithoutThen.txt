! -s SYNTACTIC
! should complain about missing then token
program p ;
begin
    if false
        a := true
    ;
end .