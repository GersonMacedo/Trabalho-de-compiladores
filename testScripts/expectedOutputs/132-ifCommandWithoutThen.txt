
Performing up to syntactic analysis at testScripts/scripts/132-ifCommandWithoutThen.txt
Parser::Expected 'then' but 'a' was found at line 6 column 9
